app/views/partials/_user_card.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class='user-card'>
+  <h3 class='profile-pic'><%= user.photo %></h3>
+  <div class='user-information'>
+    <h2 class='user-name'><%= user.name %></h2>
+    <h4 class='user-number-of-posts'>Number of posts: <%= user.postsCounter %></h4>
+  </div>
+</div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure when clicking on each **Post** takes you to its show page as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB3P7JH5WRW2BYO7YOFFAM6YY6QWGQA).